src/room_member.cpp

```diff
@@ -66,6 +66,35 @@ void RoomMember::HandleWifiPackets(const RakNet::Packet* packet) {
     }
 }
 
+
+void RoomMember::HandleRoomInformationPacket(const RakNet::Packet* packet) {
+    RakNet::BitStream stream(packet->data, packet->length, false);
+
+    // Ignore the first byte, which is the message id.
+    stream.IgnoreBytes(sizeof(RakNet::MessageID));
+
+    RakNet::RakString room_name;
+    stream.Read(room_name);
+    room_information.name.assign(room_name.C_String(), room_name.GetLength());
+    stream.Read(room_information.member_slots);
+
+    size_t num_members;
+    stream.Read(num_members);
+    member_information.resize(num_members);
+    
+    for (auto it: member_information) {
+        RakNet::RakString nick;
+        stream.Read(nick);
+        it.nickname.assign(nick.C_String(), nick.GetLength());
```

Comment:
Please use variable names which reflect the sending portion:
* `nick` -> `nickname`
* `game` -> `game_name`